compute/cker/src/train/ReduceMean.test.cc

```diff
@@ -67,7 +67,13 @@ template <typename T> class ReduceMeanVerifier
     nnfw::cker::train::MeanGrad(_out_shape, incoming.data(), _in_shape, grad.data());
 
     if (expect_eq)
-      EXPECT_EQ(grad, expected);
+    {
+      // consider the floating point error
+      for (int i = 0; i < grad.size(); ++i)
+      {
+        EXPECT_NEAR(grad[i], expected[i], 1e-3f);
+      }
+    }
```

Comment:
This is to resolve this error 

```
``` 
[2023-12-27T07:03:08.598Z] [ RUN      ] CKer_Operation.ReduceMeanGrad
[2023-12-27T07:03:08.598Z] /home/jenkins/jenkins_agent/workspace/nnfw/master/push-cross-runtime-test-debug/compute/cker/src/train/ReduceMean.test.cc:70: Failure
[2023-12-27T07:03:08.598Z] Expected equality of these values:
[2023-12-27T07:03:08.598Z]   grad
[2023-12-27T07:03:08.598Z]     Which is: { 0.499998, 0.999996, 0.499998, 0.999996 }
[2023-12-27T07:03:08.598Z]   expected
[2023-12-27T07:03:08.598Z]     Which is: { 0.5, 1, 0.5, 1 }
[2023-12-27T07:03:08.598Z] /home/jenkins/jenkins_agent/workspace/nnfw/master/push-cross-runtime-test-debug/compute/cker/src/train/ReduceMean.test.cc:70: Failure
[2023-12-27T07:03:08.598Z] Expected equality of these values:
[2023-12-27T07:03:08.598Z]   grad
[2023-12-27T07:03:08.598Z]     Which is: { 0.499998, 0.499998, 0.999996, 0.999996 }
[2023-12-27T07:03:08.599Z]   expected
[2023-12-27T07:03:08.599Z]     Which is: { 0.5, 0.5, 1, 1 }
[2023-12-27T07:03:08.599Z] [  FAILED  ] CKer_Operation.ReduceMeanGrad (0 ms
``` 
```